src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -28,14 +28,16 @@
 public final class RemoteOptions extends OptionsBase {
 
   @Option(
-      name = "remote_cache_proxy",
+      name = "remote_proxy",
+      oldName = "remote_cache_proxy",
       defaultValue = "null",
       documentationCategory = OptionDocumentationCategory.REMOTE,
       effectTags = {OptionEffectTag.UNKNOWN},
       help =
           "Connect to the remote cache through a proxy. Currently this flag can only be used to "
-              + "configure a Unix domain socket (unix:/path/to/socket) for the HTTP cache.")
-  public String remoteCacheProxy;
+              + "configure a Unix domain socket (unix:/path/to/socket) for the HTTP cache."
+              + "At the moment proxy is supported only for the remote cache, not for the remote execution.")
```

Comment:
IMO this message brings no additional value. The previous sentence makes it pretty clear.

Reply:
hm, makes sense